hadoop-hdds/dev-support/checkstyle/checkstyle.xml

```diff
@@ -71,9 +71,6 @@
     <!--<module name="FileLength">-->
     <module name="FileTabCharacter"/>
 
-    <module name="LineLength">
-        <property name="fileExtensions" value="java"/>
-    </module>
```

Comment:
Is this change intentional?

Reply:
Nope. My intellij removed it. Let me restore.